### ISSUE TITLE
Navigation keys

### ISSUE THREAD

**Issue by xogeny** [mtiller/ModelicaBook#413]
For web app, allow keyboard navigation to:
  * Return to the home page
  * Go to next topic
  * Go to next page
  * Go to parent topic
  * Give focus to search box
  * Scroll down (already there: Space)

**Comment by mtiller** [comment]
So, it turns out I just ended up adding `accesskey` attributes to a bunch of stuff and through the magic of accessibility, it all works.  The magic keys are:
- `h`: Return home
- `n`: Next topic
- `p`: Previous topic
- `u`: Parent topic
- `s`: Focus to search box
- `t`: Bring up table of contents

**Comment by mtiller** [comment]
BTW, different browsers handle this differently.  For MacOS, you need to press `Ctrl` - `Option` + *key*.  To figure out exactly how to activate the keyboard shortcuts, consult the documentation for your browser and platform (it should be under accessibility).
